File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/ServerNetworkInfoSwitchports.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Object containing switchport information.
  /// </summary>
  [DataContract]
  public class ServerNetworkInfoSwitchports {
    /// <summary>
    /// Gets or Sets _10414
    /// </summary>
    [DataMember(Name="10414", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "10414")]
    public ServerSwitchport _10414 { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ServerNetworkInfoSwitchports {\n");
      sb.Append("  _10414: ").Append(_10414).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/server_network_info_switchports.dart
part of swagger.api;

class ServerNetworkInfoSwitchports {
  
  ServerSwitchport n10414 = null;

  ServerNetworkInfoSwitchports();

  @override
  String toString() {
    return 'ServerNetworkInfoSwitchports[n10414=$n10414, ]';
  }

  ServerNetworkInfoSwitchports.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    n10414 = new ServerSwitchport.fromJson(json['10414']);
  }

  Map<String, dynamic> toJson() {
    return {
      '10414': n10414
     };
  }

  static List<ServerNetworkInfoSwitchports> listFromJson(List<dynamic> json) {
    return json == null ? new List<ServerNetworkInfoSwitchports>() : json.map((value) => new ServerNetworkInfoSwitchports.fromJson(value)).toList();
  }

  static Map<String, ServerNetworkInfoSwitchports> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ServerNetworkInfoSwitchports>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ServerNetworkInfoSwitchports.fromJson(value));
    }
    return map;
  }
}
